### DISCUSSION TITLE
Does it support Azure in China (21Vianet) data factories?

### DISCUSSION THREAD

**Discussion by lizhongyue** [Azure-Player/azure.datafactory.tools#421] [Q&A]
**Describe the bug**
##[error]The remote server returned an error: (404) Not Found.

**To Reproduce**
Your script to reproduce the behaviour:
```
21v
```
![image](https://github.com/user-attachments/assets/976c4fd5-61d8-41c4-a928-ec3aa63c8fe7)

**Comment by NowinskiK** [comment]
I'm not sure what do you mean, could you be more descriptive?

**Comment by lizhongyue** [comment]
Hi [NowinskiK](https://github.com/NowinskiK) 
Our data factory is created at https://portal.azure.cn/ (Azure China), the address of the data factory pushed when using Azure DevOps Tasks (#adftools) Publish is https://management. azure.com/, I want to push the data factory to https://management.core.chinacloudapi.cn (Azure China), can I set this address to https://management.core.chinacloudapi.cn (Azure China)? I want to push to  (Azure China), can I change this address to  (Azure China)?

**Comment by NowinskiK** [comment]
No, currently it doesn't. The workaround you can try is to download the PS module locally, change appropriate code files and import the module from this location resulting in modified code being imported.
References: https://learn.microsoft.com/en-us/azure/china/resources-developer-guide
